src/main/java/com/easypost/utils/Fixture.java

```diff
@@ -0,0 +1,390 @@
+        List<String> beginningKeys = new ArrayList<>(Arrays.asList(keyPath)).subList(0, keyPath.length - 1);
+        Map<String, Object> fixture = createFixture(beginningKeys.toArray(new String[0]));
+
+        if (fixture == null) {
+            // not a fan of this default value
+            return 0;
+        }
+        if (fixture.containsKey(lastKey)) {
+            return ((Double) fixture.get(lastKey)).intValue();
+        }
+        return 0;
+    }
+
+    private static String createFixtureString(String... keyPath) {
+        String lastKey = keyPath[keyPath.length - 1];
+        List<String> beginningKeys = new ArrayList<>(Arrays.asList(keyPath)).subList(0, keyPath.length - 1);
+        HashMap<String, Object> fixture = createFixture(beginningKeys.toArray(new String[0]));
+
+        if (fixture == null) {
+            // not a fan of this default value
+            return "";
+        }
+        if (fixture.containsKey(lastKey)) {
+            return (String) fixture.get(lastKey);
+        }
+        return "";
+    }
+
+    private static HashMap<String, Object> createFixtureMap(String... keyPath) {
+        return createFixture(keyPath);
+    }
+
+    /**
+     * Get the default page size for the API.
+     *
+     * @return The default page size for the API
+     */
+    public static int pageSize() {
+        return createFixtureInt("page_sizes", "five_results");
+    }
+
+    /**
+     * Get the default USPS carrier account ID.
+     *
+     * @return The default USPS carrier account ID
+     */
+    public static String uspsCarrierAccountID() {
+        // Fallback to the EasyPost Java Client Library Test User USPS carrier account
+        return System.getenv("USPS_CARRIER_ACCOUNT_ID") != null ? System.getenv("USPS_CARRIER_ACCOUNT_ID") :
+                "ca_f09befdb2e9c410e95c7622ea912c18c";
+    }
+
+    /**
+     * Get the USPS carrier name.
+     *
+     * @return The USPS carrier name
+     */
+    public static String usps() {
+        return createFixtureString("carrier_strings", "usps");
+    }
+
+    /**
+     * Get the default USPS service level.
+     *
+     * @return The default USPS service level
+     */
+    public static String uspsService() {
+        return createFixtureString("service_names", "usps", "first_service");
+    }
+
+    /**
+     * Get the default pickup service level.
+     *
+     * @return The default pickup service level
+     */
+    public static String pickupService() {
+        return createFixtureString("service_names", "usps", "pickup_service");
+    }
+
+    /**
+     * Get the default pickup date.
+     *
+     * @return The default pickup date
+     */
+    public static String pickupDate() {
+        /*
+        If you need to re-record cassettes, increment the date below and ensure it is one day in the future,
+        USPS only does "next-day" pickups including Saturday but not Sunday or Holidays.
+         */
+        return "2022-08-30";
+    }
```

Comment:
Nit: would prefer this just be in the `pickup` fixture like other libs so someone doesn't need to go finding this since it will be the only one differing from this pattern.

Reply:
Resolved.